[BUG] - Issues with Config Validation testing suite

### What bug did you encounter?

Some of the output given by the `Config Validation` test for the Assertion error for at least 3 responses in `noSelfXP` and `noBotXP` gave erroneous output as - 

>  1) Config Validation
       Becca Responses
         noSelfXP has at least three values:
     AssertionError: noSelfXP only has 8 values
      at Context.<anonymous> (test/config.spec.ts:12:15)
      at processImmediate (node:internal/timers:464:21)

  2) Config Validation
       Becca Responses
         noBotXP has at least three values:
     AssertionError: noBotXP only has 7 values
      at Context.<anonymous> (test/config.spec.ts:12:15)
      at processImmediate (node:internal/timers:464:21)
      
While they both had only 2 values.

### What did you expect to happen?

The error output to say - 

>  1) Config Validation
       Becca Responses
         noSelfXP has at least three values:
     AssertionError: noSelfXP only has 2 values
      at Context.<anonymous> (test/config.spec.ts:12:15)
      at processImmediate (node:internal/timers:464:21)

  2) Config Validation
       Becca Responses
         noBotXP has at least three values:
     AssertionError: noBotXP only has 2 values
      at Context.<anonymous> (test/config.spec.ts:12:15)
      at processImmediate (node:internal/timers:464:21)

### Screenshots

![Screenshot 2021-12-16 at 8 00 52 PM](https://user-images.githubusercontent.com/62864373/146404349-cd00f589-2fbc-42da-9556-5fe09696287f.png)



### Additional information

_No response_

--- a/test/config.spec.ts
+++ b/test/config.spec.ts
@@ -9,7 +9,7 @@ suite("Config Validation", () => {
     for (const key in BeccaPhrases) {
       const value = BeccaPhrases[key];
       test(`${key} has at least three values`, () => {
-        assert(value.length >= 3, `${key} only has ${key.length} values`);
+        assert(value.length >= 3, `${key} only has ${value.length} values`);
       });
 
       for (const response of value) {
@@ -27,7 +27,7 @@ suite("Config Validation", () => {
     for (const key in BeccaSass) {
       const value = BeccaSass[key];
       test(`${key} has at least three values`, () => {
-        assert(value.length >= 3, `${key} only has ${key.length} values`);
+        assert(value.length >= 3, `${key} only has ${value.length} values`);
       });
 
       for (const response of value) {

